ui/toast.go

```diff
@@ -37,14 +37,19 @@ func displayToast(th *decredmaterial.Theme, gtx layout.Context, n *toast) layout
 	})
 }
 
-// Timer implements the standard time.AfterFunc. It only creates a new time.Timer when toast.timer is nil.
+// Timer is only creates a new time.Timer when toast.timer is nil.
 // Gio re-renders its UI recursively and Timer prevents multiple time.Timer from being created before the duration is
 // exceeded.
 func (n *toast) Timer(d time.Duration, f func()) {
```

Comment:
This method can be removed. Using time.AfterFunc would be sufficient. The entire toast.go file can be deleted and `displayToast` method can be added to components.go